index.js

```diff
@@ -13,30 +13,69 @@ module.exports = function(callback, options) {
   options = {
     $current: options.$current || 1,
     max: options.max,
-    timeout: options.timeout || undefined
+    timeout: options.timeout || undefined,
+    match: options.match || [],
+    stepbackBase: options.stepbackBase === undefined ? 1.2 : options.stepbackBase,
+    stepbackExponent: options.stepbackExponent || 1.5
```

Comment:
If the default for exponential backoff is not 0 this will likely have to be a major bump (but that's probably ok)